### PR TITLE
Add default page

### PR DESCRIPTION
**Pull request by osfameron**
Reported by David Elliott, thanks!

  localhost:8080/

Now gives a bare HTML page with links to the apidocs, as per other sample apps.

### CODE REVIEW

File: try-cb-dotnet/wwwroot/index.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<h1>.NET Travel Sample API</h1>
+    A sample API for getting started with Couchbase Server and the .NET SDK.
+    <ul>
+        <li><a href="/apidocs">Learn the API with Swagger, interactively</a>
+        <li><a href="https://github.com/couchbaselabs/try-cb-dotnet">GitHub</a>
+    </ul>
```
Comment (maria-robobug) [inline try-cb-dotnet/wwwroot/index.html]: Probably good to have newline here.

**Review by maria-robobug — APPROVED** [review]
👍 LGTM, just a small comment.

File: try-cb-dotnet/wwwroot/index.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<h1>.NET Travel Sample API</h1>
+    A sample API for getting started with Couchbase Server and the .NET SDK.
+    <ul>
```
Comment (maria-robobug) [inline try-cb-dotnet/wwwroot/index.html:3]: Indentation seems weird here actually, maybe something like:
```
<h1>.NET Travel Sample API</h1>
A sample API for getting started with Couchbase Server and the .NET SDK.
<ul>
   <li><a href="/apidocs">Learn the API with Swagger, interactively</a>
   <li><a href="https://github.com/couchbaselabs/try-cb-dotnet">GitHub</a>
</ul>
```

File: try-cb-dotnet/wwwroot/index.html (hunk 1 above)
Comment (osfameron) [reply try-cb-dotnet/wwwroot/index.html]: At the end of the file, you mean? I suspect I'm using the VSCode defaults, is that worth changing?

Comment (maria-robobug) [reply try-cb-dotnet/wwwroot/index.html]: yeah, I always set it to automatically add a newline at the end of a file, just because I forget all the time 😄

Comment (osfameron) [reply try-cb-dotnet/wwwroot/index.html]: ok, updated settings, force pushed.

**Pull request merged**